lua/astrocommunity/pack/typescript/README.md

```diff
@@ -8,6 +8,7 @@ This plugin pack does the following:
 - Adds `eslint_d` linter
 - Adds [JSON language support](../json)
 - Adds support for dap for JS
+- Add [tsc.nvim](dmmulroy/tsc.nvim) to run type-checking on `:TSC`
```

Comment:
We should have this in another PR. Just to make this PR only about typescript-tools.